crates/hyperswitch_connectors/src/connectors/cybersource/transformers.rs

```diff
@@ -1349,7 +1402,22 @@ impl
                 )
                 .ok();
                 let effective_authentication_type = authn_data.authentication_type.map(Into::into);
-
+                let network_score: Option<u32> =
+                    if ccard.card_network == Some(common_enums::CardNetwork::CartesBancaires) {
+                        authn_data
+                            .message_extension
+                            .as_ref()
+                            .and_then(|secret| {
+                                serde_json::from_value::<Vec<MessageExtensionAttribute>>(
+                                    secret.clone().expose(),
+                                )
+                                .ok()
```

Comment:
We are ignoring the deserialization error here (and elsewhere in this file). Is that intentional?

Reply:
@SanchithHegde yes this was intentional, but thanks for the suggestion - added the deserialization logs for unexpected errors